packages/comparisons/src/data.json

```diff
@@ -18346,6 +18494,19 @@
 			"preset": "untyped"
 		}
 	},
+	{
+		"eslint": [
+			{
+				"name": "@typescript-eslint/no-useless-default-assignment",
+				"url": "https://typescript-eslint.io/rules/no-useless-default-assignment"
+			}
+		],
+		"flint": {
+			"name": "uselessDefaultAssignments",
```

Comment:
I can't think of a naming pattern for this & similar rules that would put "default assignments" before "useless". But if you can, it'd be nice to use that.

Reply:
Yeah. For now I suppose we can go with this, and eventually maybe think of something more Flint-like...